presenting-tosw.adoc

```diff
@@ -1,41 +1,80 @@
+So from this introduction you should take two important points:
+
+. There is a _what_, a _how_, and a _why_ to the practice of successfully and sustainably creating and maintaining open source software.
+. This guide distills opinions from a sizeable group of open source practitioners, the kind of people always having to answer the question of _why we do this open source thing the way we do it_.
+
+In particular, this guide is a living document that endeavors to gather and spread a diverse group of voices who are similarly (and somewhat-opinionatedly) collecting the best ways to create and manage an open source software community.
 
 These voices are all part of https://theopensourcway.org[this community of practice around open source software community management].
 
-=== Assumptions we are making
+== Assumptions we're making
+
+When you look at the table of contents for this guide, notice how the flow of focus goes from the user, to the participant, to the contributor.
+
+This arrangement specifically tracks our view of how a community comes about and grows. And it all starts with a need.
+
+. A person or persons needing a solution creates the first iteration of the software.
+. Other people with similar needs start using it, forming the core of the userbase.
+. From the userbase arise people who are enthusiasts of the software and things about/around it.
+. Finally arising from the users and the enthusiasts come people who are interested and have grown capable of contributing to the software in some way.
+
+Thus, the primary and continual focus for a project is to _take good care of your users_, for some of them will become your contributors, and you want them to do it out of love more than anger.
 
-'Details of how we see growing a community via growing users, lowering barriers, and enabling people to participte and contribute at all levels. How the guide reflects this structure.'
+A key element of all this is recognizing the value in contributions of all types, at all levels.
+Rather than valorizing only a single type of contribution, the best practice is to treat all manner of gifts to the project as equal.
+These contributions comprise content and code, hours and effort, and all manner of broad contributons for the project via events, discussion forums, infrastructure maintenance, and so on.
 
-=== Structure of this guide
+When you lower barriers to participation, you make your project welcoming at all levels.
+This doesn't mean you _remove all barriers_—just lower most of them appropriately.
+Appropriate barriers are necessary for someone receiving keys to systems, for example, and these barriers are different from those allowing only certain participants to contribute a help article or a patch to the configuration management system.
+
```

Comment:
I feel like this section just sort of ... stops. Maybe it would benefit from some sort of closing sentiment?

Reply:
Should be fixed in the current merging of my messes